{"uuid": "6f602839-a681-4210-a43e-bf663bfaf988", "children": ["e17fcfc7-5ac9-4077-90cd-acbd37772067"], "befores": [{"name": "_Class__pytest_setup_class", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 87\nCurrent browser version is 91.0.4472.124 with binary path C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe\n\n", "trace": "  File \"c:\\python38\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\fixtures.py\", line 915, in call_fixture_func\n    fixture_result = next(generator)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\python.py\", line 745, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"c:\\python38\\lib\\site-packages\\_pytest\\python.py\", line 674, in _call_with_optional_argument\n    func(arg)\n  File \"E:\\前端自动化\\YuAutotest\\testcase\\test_login.py\", line 14, in setup_class\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 76, in __init__\n    RemoteWebDriver.__init__(\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"c:\\python38\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1626164370372, "stop": 1626164373556}], "start": 1626164370372, "stop": 1626164373734}